src/main.rs

```diff
@@ -24,6 +27,13 @@ struct Cli {
 enum Commands {
     #[clap(name = "upload")]
     Upload {
+        #[arg(
+            long,
+            required = false,
+            value_delimiter = ' ',
+            help = "Test command to invoke."
+        )]
+        command: Vec<String>,
```

Comment:
We can either have this as a specific arg or as a catch all for all parameters after the upload command. I leaned more towards a specific `command` arg since it is more clear what the intent is. The other option is to have a separate `test` command that mimics the `upload` command but also takes in a command. Thoughts?

Reply:
Opted to just have a separate `test` command.